invenio_records_resources/factories/factory.py

```diff
@@ -38,9 +38,7 @@ class RecordTypeFactory(object):
     service_config_cls = None
     service_cls = None
 
-    _schema_path_template = (
-        "https://localhost/schemas/{name_plural}/{name}-v{version}.json"
```

Comment:
Comment: I'm unsure what impact this might have. The correct is ``local://`` just not sure if there's any impact downstream.

Reply:
I checked out these PRs and the records are not being created anymore, could be due to this change?

Reply:
In principle this should only affect tests because each of the defined model should have a proper schema definition (otherwise a schema couldn't be loaded.